Simplified/WelcomeScreen/OETutorialChoiceViewController.swift

```diff
@@ -112,6 +112,17 @@ class OETutorialChoiceViewController : UIViewController {
     let libAccount = AccountsManager.shared.currentAccount
     let userAccount = NYPLUserAccount.sharedAccount()
     if libAccount?.details == nil {
+      
+      if (libAccount == nil) {
```

Comment:
should this `if` be moved outside of the `if libAccount?.detail`? Seems more like a `guard` to be placed at the top?

Reply:
It looks to me it's trying to load the catalog directly, maybe because we are not retaining/getting the library account info?
If this is temporary workaround, can @singhraman4282 put a note on it?
Also, we should find out what's the underlying issue here.

Reply:
All of a sudden the issue stopped happening. I was unable to reproduce this upon multiple attempts.